src/main/resources/cloudformation/vpc.yaml

```diff
@@ -43,6 +47,10 @@ Parameters:
     AllowedPattern: '[a-z]{2}-[a-z]+-\d\w'
     Description: An availability zone identifier for zone '3'
     Type: String
+  az4:
+    Default: ""
```

Comment:
Seems like it's missing allowed pattern

Reply:
So this was something I was having a problem with. If the allowed pattern is included, this parameter cannot be an empty string. I wasn't sure what would be a good way to include the pattern while also allowing this to be an empty string.

Reply:
That's fine then